src/main/java/com/hibernate/dao/OrderDao.java

```diff
@@ -0,0 +1,13 @@
+package com.hibernate.dao;
+
+import com.hibernate.model.Order;
+import com.hibernate.model.User;
+import java.util.List;
+
+public interface OrderDao {
+    Order add(Order order);
+
+    List<Order> getOrdersByUser(User user);
+
+    List<Order> getAll();
```

Comment:
do we need this method in our task?
Hibernate-04 Checklist
Do not create redundant functionality, like methods, that are not required by task
I think the same must be to the whole this project

Reply:
At this point we can remove all "getAll" methods.I think it`s basic method and at last tasks we implement it without any doubt)

Reply:
let's remove this. Without any doubts —not an argument)